playbooks/common/openshift-cluster/upgrades/v3_1_to_v3_2/upgrade.yml

```diff
@@ -16,14 +45,20 @@
 ###############################################################################
 - name: Upgrade master
   hosts: oo_masters_to_config
+  vars:
+    openshift_master_ha: "{{ groups.oo_masters_to_config | length > 1 }}"
```

Comment:
chances are you may need to fall back to computing it yourself if it isn't available, but if it is there we should respect it (since it would technically allow for a master to be treated as an HA master).